### PR TITLE
Remove new design logic

### PR DESCRIPTION
**Pull request by hannako**
## What

Remove unused templates and css files following https://github.com/alphagov/frontend/pull/3835

## Why

- [Trello card](https://trello.com/c/IhFaIzdI/2220-remove-the-now-redundant-newdesign-code-from-frontend-m), [Jira issue NAV-12392](https://gov-uk.atlassian.net/browse/NAV-12392)

## Follow up work

- In a fresh PR in this application we should rename the files prefixed with `_new` 
   -  app/views/homepage/_more_on_govuk_new.html.erb
   - app/assets/stylesheets/views/_homepage_more_on_govuk_new.scss
   - app/views/homepage/_promotion_slots_new.html.erb
   - app/assets/stylesheets/views/_homepage_new.scss
   - image_src_new

- We also need to remove the env var from helm charts by reverting https://github.com/alphagov/govuk-helm-charts/pull/1316

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -611,17 +610,14 @@ en:
         - text: Find out payment dates and how to report a payment is missing.
           title: Cost of Living Payment
           href: /guidance/cost-of-living-payment
-          image_src: homepage/cost-of-living-november-2022.jpg
           image_src_new: homepage/cost-of-living.png
```
Comment (patrickpatrickpatrick) [inline config/locales/en.yml:613]: Could these be renamed to image_src now since we don't have both image_source and image_source_new?

Comment (patrickpatrickpatrick) [reply config/locales/en.yml:613]: Oh I see that's in the PR description, never mind

Comment (hannako) [reply config/locales/en.yml:613]: I made a note in the follow up work section of the PR descriptions as there are quite a few that have a `_new` prefix.

Comment (hannako) [reply config/locales/en.yml:613]: I can do it now if you think that's better?

Comment (patrickpatrickpatrick) [reply config/locales/en.yml:613]: Actually I think it's fine splitting this up into multiple PRs because it'll be easier to review

**Review by patrickpatrickpatrick — APPROVED** [review]

**Review by andysellick — APPROVED** [review]
Analytics looks correct 👍

**Pull request merged**